### ISSUE TITLE
Github Actions and Travis builds need CVC4

### ISSUE THREAD

**Issue by robdockins** [GaloisInc/cryptol#750]
The Cryptol test suite now uses CVC4 in one of its tests.  Some (but not all) of the CI checks for this repository need to be updated to include a recent CVC4 to make the test suite go green.

**Comment by robdockins** [comment]
CF #699

**Comment by jared-w** [comment]
I actually anticipated this and have already built the CI pipeline to support CVC4 because I'm cool like that. However, shortly after I did so, I noticed that the only supported way to install CVC4 on macos is through the homebew tap which compiles it from source. This takes 15 minutes and doesn't seem to be cacheable, which is a bit annoying, but isn't the end of the world.

I'll write a small PR to enable CVC4 on macos and then work on seeing if I can cache the homebrew install somehow.
